src/Razor/src/Microsoft.AspNetCore.Razor.Language/Syntax/SyntaxNode.cs

```diff
@@ -224,6 +224,35 @@ internal SyntaxToken GetFirstToken()
             return ((SyntaxToken)GetFirstTerminal());
         }
 
+        internal SyntaxList<SyntaxToken> GetTokens()
```

Comment:
Does it need to be internal? Aren'tt hese types internal?

Reply:
I thought the same thing but just followed the same pattern as everything else in this class.